contrib/skopeo/template.py

```diff
@@ -0,0 +1,49 @@
+pkgname = "skopeo"
+pkgver = "1.6.0"
+pkgrel = 0
+build_style = "go"
+# for compatibility with Makefile targets
+make_dir = "bin"
+make_build_args = [
+    "-mod",
+    "vendor",
+    "./cmd/skopeo",
+]
+hostmakedepends = [
+    "bash",
+    "gmake",
+    "go",
+    "go-md2man",
+    "pkgconf",
+]
+makedepends = [
+    "device-mapper-devel",
+    "gpgme-devel",
+    "libbtrfs-devel",
+    "linux-headers",
+]
+depends = [
+    "containers-common",
+]
+pkgdesc = "OCI image and repo manipulation tool"
+maintainer = "Radosław Piliszek <radek@piliszek.it>"
+license = "Apache-2.0"
+url = "https://github.com/containers/skopeo"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "95d63d786e7efda7711fbbad6e37edf53ac767eaa47a63dc56c19bbd95add2cd"
+# NOTE(yoctozepto): assumes awk is gawk; requires docker; probably more...
+options = ["!check"]
```

Comment:
with bash added back though you can do

```suggestion
def do_check(self):
    self.do("gmake", "test-unit-local")
```

Reply:
yeah just throw this in for the few unit tests and this is ready

Reply:
I'm looking into it now. Thanks.

Reply:
Done. I have checked the other tests once again and adapted the comment accordinly..

Reply:
thanks! i put it in the correct ordering (the pre/do/post are ordered based on the actual step order) for you

Reply:
Thanks.